packages/datx-jsonapi-angular/projects/datx-jsonapi-angular/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@datx/jsonapi-angular",
+  "version": "2.3.1",
+  "peerDependencies": {
+    "@angular/common": ">=12.2.0",
+    "@angular/core": ">=12.2.0",
+    "@datx/core": "^2.3.1",
```

Comment:
I missed this before. The peer dependency for all datx packages should be 2.x.

Reply:
I am not 100% sure it will work with e.g. 2.0.0, will check

Reply:
2.x is the same as >=2, so it covers 2.0.0 and upwards.

Reply:
You could put >=2.3.0 then.

Reply:
That's basically the same as `^2.3.0`, should I put that?

Reply:
Ah, you are right 🤦‍♂️  Let's leave it like this then.